Frameworks/UIKit/UIView.mm

```diff
@@ -1405,10 +1395,8 @@ - (void)setFrame:(CGRect)frame {
                      frame.size.height);
     }
 
-    CGRect startFrame = frame;
-
-    if (frame.origin.x == doRound(curFrame.origin.x) && frame.origin.y == doRound(curFrame.origin.y) &&
-        frame.size.width == doRound(curFrame.size.width) && frame.size.height == doRound(curFrame.size.height)) {
+    // Don't bother doing work if there's no work to be done
+    if (memcmp(&frame, &curFrame, sizeof(CGRect)) == 0) {
```

Comment:
probably not worth worrying about right now; this is existing code moved down from above to avoid two checks.  i suppose we could just keep the distinct field checks instead, but the memcmp seems fine to me.

---
In reply to: [99915102](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r99915102) [](ancestors = 99915102)